styles/globals.css

```diff
@@ -20,3 +20,28 @@ a {
 * {
   box-sizing: border-box;
 }
+
+/* styles for top under construction banner start here*/
+
+.banner {
+  background: white;
+}
+
+.banner>div {
+  padding: 1rem;
```

Comment:
Same [comment](https://github.com/reactdeveloperske/reactdevske-website/pull/63/files#r900877750) applies here

Reply:
changed these in commit